code/modules/mob/living/carbon/human/species.dm

```diff
@@ -368,7 +406,7 @@
 	poison_type = "phoron"//"oxygen"
 	insulated = 1
 
-	flags = NO_SCAN
```

Comment:
Again, how is this related, if it's because you added a species pick system just make a special flag like IS_BLACKLISTED instead of adding whitelists for everything, seems hacky.

Reply:
Good point, sorry.